src/Lumi/Components/Textarea.purs

```diff
@@ -77,6 +77,26 @@ defaults =
   , value: ""
   }
 
+textareaPlaceholderText_ :: String -> TextareaProps
+textareaPlaceholderText_ s =
```

Comment:
I'm not sure we need a separate function for this - isn't it just `Textarea.defaults { placeholder = s }` ?

Reply:
Likewise I was suggesting removing this function entirely, not just simplifying the implementation.

Reply:
Removed.